plugins/include/open62541/plugin/nodestore_default.h

```diff
@@ -2,6 +2,7 @@
  * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
  *
  *    Copyright 2019 (c) Julius Pfrommer, Fraunhofer IOSB
+ *    Copyright 2020 (c) Kalycito Infotech Pvt Ltd
```

Comment:
Remove, as no real changes were made in this file.

Reply:
@jpfr Thanks for the comments.
Can we move the lookupTable structure, encode and decode function declarations to this (nodestore_default.h) file?